projects/tn-ro-manual.diff

```diff
@@ -1250,7 +1253,7 @@ index 8ff8b2a..b3745d3 100644
  					}
  				}
  			}
-@@ -6671,10 +6735,10 @@ Es werden Referenzen (`NetworkReference`) zu allen Netzwerk-Elementen (z.B. `Roa
+@@ -6673,10 +6737,10 @@ Es werden Referenzen (`NetworkReference`) zu allen Netzwerk-Elementen (z.B. `Roa
```

Comment:
same here. Will not comment for every case now, seems to be a strucural issue probably when creating the auto diff at is already incomplete there?